### ISSUE TITLE
Add Contact Tab

### ISSUE THREAD

**Issue by bvanderdrift** [AreaFiftyLAN/lancie-frontend#359]
There is a contact page at https://areafiftylan.nl/contact which can only be reached through hrefs within the text. I'd say let's add it as a tab on the homepage.

**Comment by martijnjanssen** [comment]
Good point. We will not add this to the header though. We are looking to change the position of the links to be more suitable for expansions like this.

**Comment by martijnjanssen** [comment]
We'll add this to the footer.
